### DISCUSSION TITLE
evento.redattoNellaCasaComunale (Nascita)

### DISCUSSION THREAD

**Discussion by OrazioCostanzo** [italia/ansc#1103] [Supporto servizi cooperativi]
Salve,
si puo' avere una spiegazione dettagliata sul significato di questo flag? quando deve essere impostato a true e quando a false? grazie.

**Comment by acolosimosogei** [comment]
Salve, 

il flag "evento.redattoNellaCasaComunale" nelle tabelle di "Mapping_casi_uso" viene utilizzato sempre con il valore "evento.redattoNellaCasaComunale,!=,false" e si trova sotto la colonna "Condizioni obbligatorietà" il che va letto come: 'se l'evento viene redatto nella casa comunale la sezione relativa può essere resa non visibile'.

Inoltre le sezioni relative a quel flag sono sempre non obbligatorie.

Grazie e buona giornata.
Assistenza ANSC

**Comment by acolosimosogei** [comment]
Buongiorno, 

procedo a chiudere la discussion.

Grazie,
Assistenza ANSC
